stack-8.10.1.yaml

```diff
@@ -1,12 +1,12 @@
-resolver: nightly-2020-06-15
-compiler: ghc-8.10.1
+resolver: nightly-2020-06-29
 
 packages:
 - .
 - ./ghcide/
 
 extra-deps:
-- Cabal-3.2.0.0
+# - Cabal-3.2.0.0
+- Cabal-3.0.2.0
```

Comment:
Why is that necessary?

Reply:
stack wouldn't build otherwise, and the cabal plan.json had that.   Except it just struck me that was the 8.8.3 plan.json.

Let me double check.

Reply:
Why do we need Cabal in the first place again? Is there some plugin that uses it?